### ISSUE TITLE
no 5 minute data

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/pvnet_app#41]
## Describe the bug
When there is no 5 minute data, then pvnet fails over. 

## To Reproduce
https://www.notion.so/openclimatefix/2024-02-07-Missing-file-5b8fed94aec74fa8838500e53f4c8d39?pvs=4

## Expected behavior
moves on to the 15 minute satellite data automatically

**Comment by peterdudfield** [comment]
Need to check if the file exsits here - https://github.com/openclimatefix/pvnet_app/blob/main/pvnet_app/data.py#L27

and adjust logic here - https://github.com/openclimatefix/pvnet_app/blob/main/pvnet_app/data.py#L43

**Comment by dfulu** [comment]
No, at the minute we don't train with dropout, but we do have the support for it

**Comment by peterdudfield** [comment]
Now working on `0.0.32`
